simpleclient/src/main/java/io/prometheus/client/Reservoir.java

```diff
@@ -0,0 +1,85 @@
+package io.prometheus.client;
+
+import java.util.Arrays;
+import java.util.concurrent.ThreadLocalRandom;
```

Comment:
This is a Java 7 feature, this library is targeted at Java 6 as 2/3 of our users still use Java 6. You'll need to find a way to do this within the Java 6 featureset.